code/js/rcalc-constant-html-class.js

```diff
@@ -0,0 +1,6 @@
+// classes
```

Comment:
For constants you should create new folder: constant.

Reply:
folder has been created